cleanlab_studio/cli/dataset/schema_helpers.py

```diff
@@ -148,10 +150,18 @@ def multiple_separate_words_detected(values: Collection[Any]) -> bool:
 
 def is_filepath(string: str, check_existing: bool = False) -> bool:
     if check_existing:
-        return os.path.exists(string)
+        return pathlib.Path(string).exists()
     return pathlib.Path(string).suffix != "" and " " not in string
 
 
+def is_url(string: str) -> bool:
```

Comment:
is there a reason we're not doing a `check_existing` here? seems rational to me that the default check is just seeing if it's a valid URL, and if we want to check that it exists, we can do the `head` call

Reply:
That was my initial approach, but I was having trouble figuring out how to check if the string was a valid url in a way that wouldn't include certain filepaths as valid urls (i.e. filepath starting with C:// or s3://). Do you have any ideas here?

Reply:
Looks like this thread might have some options: https://stackoverflow.com/questions/7160737/how-to-validate-a-url-in-python-malformed-or-not